angular/core/components/thread_page/thread_page_controller.coffee

```diff
@@ -32,9 +32,8 @@ angular.module('loomioApp').controller 'ThreadPageController', ($scope, $routePa
   @init = (discussion) =>
     if discussion and !@discussion?
       @discussion = discussion
-      if @discussion.hasActiveProposal() and $location.search().proposal == @discussion.activeProposal().key
-        @proposalToFocus = @discussion.activeProposal()
-
+      @activeProposalKey = $routeParams.proposal or $location.search().proposal
+      @activeCommentId   = parseInt($routeParams.comment or $location.search().comment)
```

Comment:
This lets us take our comment id from the route params `d/abc/comment/123`, or `?comment=123` (so we can be backwards compatible)